src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -132,7 +132,17 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
             <div className="profile-extras">
               {renderedProfileQuestionAnswers}
             </div>
+            {/* Proper styling + add editing possibility */}
             <div>{renderedProfileLinks}</div>
+            {selectedUserProfile.realName && (
+              <p>{selectedUserProfile.realName}</p>
+            )}
+            {selectedUserProfile.companyDepartment && (
+              <p>{selectedUserProfile.companyDepartment}</p>
+            )}
+            {selectedUserProfile.companyTitle && (
+              <p>{selectedUserProfile.companyTitle}</p>
+            )}
```

Comment:
[nit] May as well destructure these like all of the other values here?

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1667/commits/3da671cd41b87e2e89594ea2e62bcb2fb7fde73d